go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/mongodb/mongodb-atlas-cli/atlascli
 
-go 1.23.5
-
-toolchain go1.23.6
```

Comment:
if they both define a full version you don't need both, you only need toolchain if you define go like `go 1.23`